docs/embedding/static-embedding.md

```diff
@@ -8,9 +8,9 @@ redirect_from:
 
 Also known as: static embedding.
 
-In general, embedding works by displaying a Metabase URL inside an iframe on your website. A **signed embed** (or static embed) is an iframe that's secured with a signed JSON Web Token (JWT). The signed JWT will prevent website visitors from poking around in your Metabase through the iframe.
+In general, embedding works by displaying a Metabase URL inside an iframe on your website. A **static embed** (or signed embed) is an iframe that's secured with a signed JSON Web Token (JWT). The signed JWT will prevent website visitors from poking around in your Metabase through the iframe.
 
-Signed embeds can't be used with [data sandboxes](../permissions/data-sandboxes.md) or [auditing tools](../usage-and-performance-tools/audit.md) because signed JWTs don't create user sessions (server-side sessions).
+You can can't use static embeds with [data sandboxes](../permissions/data-sandboxes.md) or [auditing tools](../usage-and-performance-tools/audit.md) because signed JWTs don't create user sessions (server-side sessions).
```

Comment:
Maybe worth mentioning that there is no drill-down, too?

Reply:
Good opportunity to suggest looking at interactive embedding, too, which supports these features?